packages/react-button/src/components/Button/Button.test.tsx

```diff
@@ -1,18 +1,18 @@
 import * as React from 'react';
-import { Button } from './Button';
+import { Button as BaseButton } from './Button';
 import * as renderer from 'react-test-renderer';
 import { mount, ReactWrapper } from 'enzyme';
 import { isConformant } from '../../common/isConformant';
+import { withThemeProvider } from '@fluentui/react-theme-provider';
 
-import { MergeStylesProvider } from '@fluentui/react-theme-provider';
-import { ButtonProps } from './Button.types';
+/** Use a ThemeProvider wrapper around the component to ensure styles show up in snapshots. */
+const Button = withThemeProvider(BaseButton);
```

Comment:
i think we should create helper in test-utils to handle this in `renderer.create`